Calorie count changes and adding number of serves

josef-butler opened the pull request:
Hey @apurv-1 here is my PR to close #6. As mentioned on the issue, changes made are:

- 'grams' has been removed from the calorie amount
- Number of serves that a recipe makes has been added
- The calorie count has been updated to reflect a single serve rather than the whole recipe

Commit:
Add number of serves and make calorie count relate to a single serve instead of full recipe

apurv-1 reviewed (commented):
Seems good! It would be great if you could attach some screenshots here too.
And I don't think `package-lock.json` has to do anything with the changes made. So, can you unstage that file?

Commit:
Revert "Add number of serves and make calorie count relate to a single serve instead of full recipe"

This reverts commit 5f1798b0a8c330e10a437380de861ca8ddbece90.

Commit:
Add number of serves and calorie field improvements. Also, change p tags to span to stop console errors

josef-butler commented:
Hey @apurv-1 yup no worries! I've updated the PR to remove `package-lock.json` - sorry about that, I've seen it included in PRs previously so kept it, but I agree it doesn't relate to this set of changes.

I've also changed a couple of `<p>` tags to `<span>` as the browser was throwing errors having `<div>` tags embedded in them.

Here are a couple of screenshots to show the new behaviour:

![foodrecipe-after](https://user-images.githubusercontent.com/48168036/136712459-3a179149-3a49-4ad9-a4df-88eddd754c63.png)

![foodrecipe-after-2](https://user-images.githubusercontent.com/48168036/136712461-69b390f1-f516-45dc-a21c-9e2fec436551.png)

apurv-1 reviewed (approved):
Looks Good, Thanks for the contribution!
